docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -2,14 +2,14 @@
 Adding a New Node Into an Existing Scylla Cluster (Out Scale)
 ==============================================================
 
-When you add a new node, another node in the cluster streams data to the new node. This operation is called bootstrapping and may
+When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
 be time-consuming, depending on the data size and network bandwidth. If using a :ref:`multi-availability-zone <faq-best-scenario-node-multi-availability-zone>`, make sure they are balanced.
 
 
 Prerequisites
 -------------
 
-Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command.
+#. Before adding the new node, check the node's status in the cluster using :doc:`nodetool status </operating-scylla/nodetool-commands/status>` command.
 You cannot add new nodes to the cluster if any nodes are down.
 
 For example:
```

Comment:
```suggestion
    For example:
```

Reply:
I can't leave the suggestion fot the following code block, but you should add four spaces at the beginning of each line.

```
    .. code-block:: shell

       Datacenter: DC1
       Status=Up/Down
       State=Normal/Leaving/Joining/Moving
       --  Address        Load       Tokens  Owns (effective)                         Host ID         Rack
       UN  192.168.1.201  112.82 KB  256     32.7%             8d5ed9f4-7764-4dbd-bad8-43fddce94b7c   B1
       DN  192.168.1.202  91.11 KB   256     32.9%             125ed9f4-7777-1dbn-mac8-43fddce9123e   B1
```